creator-node/src/diskManager.ts

```diff
@@ -525,16 +521,19 @@ async function _copyLegacyFiles(
         copiedPaths.push({ legacyPath, nonLegacyPath })
       } else {
         erroredPaths[legacyPath] = e.toString()
+        const errWithoutPaths = replaceAll(
```

Comment:
ah well the problem is that that specific error message isn't chosen by us. I think that one in particular is fs.copyFile or something similar, which comes in full form of `Error: ENOENT: no such file or directory, copyfile 'whatever path1' -> 'whatever path2'`. and if it's some other unexpected error it could also include the path so we have to make sure that path doesn't make into prometheus to spam the cardinality

Reply:
fair - [my above comment applies here too](https://github.com/AudiusProject/audius-protocol/pull/4468#discussion_r1047513559)